### ISSUE TITLE
Autocomplete also from user dictionary

### ISSUE THREAD

**Issue by velmafia** [ribtoks/xpiks#615]
It will be nice if autocomplete mechanism will suggest keyword also from user dictionary.

**Comment by ribtoks** [comment]
Good idea, but requires some dances with [libface](https://github.com/ribtoks/cpp-libface). Not sure I can jump on it right away, but I'll let it mature in the backlog for now.
